app/views/admin/aid_applications/index.html.erb

```diff
@@ -22,7 +22,7 @@
         <% @aid_applications.each do |aid_application| %>
           <tr id="<%= dom_id(aid_application) %>">
             <td><%= aid_application.id %></td>
-            <td><%= link_to aid_application.application_number, edit_organization_aid_application_eligibility_path(aid_application.organization, aid_application) %>
+            <td><%= link_to aid_application.application_number, organization_aid_application_path(aid_application.organization, aid_application) %>
```

Comment:
so admins will go to the same view as assisters if they click an application? that seems like a reasonable default.

Reply:
Yes. The story was a little under-specified about where admins should go. The Submission tab has a lot of info and it felt weird to bump the admin to action a supervisor might take.

Thinking more, maybe it should be slightly smarter, like bumping the admin to submission or finish.